dedicated-community-leader.md

```diff
@@ -38,42 +38,28 @@ Community work - especially communication - make up for a significant percentage
 If a community can not grow fast enough and pick up enough speed, chances are they won't be able to convincingly demonstrate the potential of InnerSource.
 
 **Review comments**
-- (**Done**) managers won't commit experienced developers as a community leader if they are not yet sold on InnerSource and as long as there are other, important projects 
-- (**Done**) state forces in a way that clarifies how they make the problem harder. Think constraints.
 - (**Open**: I have added something to the problem statement instead) now added that to the maybe reference pattern tbd (start small, experiment then scale up as it has proven successful)
-- (**Done**) clarify role and influence of companies cultural fit for InnerSource 
 
 ## Solution
 
-Select a community leader who 
+Select a community leader who:
 - is experienced in the Open Source working model or similar community based working models, 
 - has the required soft-skills to act as a natural leader,
 - is an excellent networker and who
 - inspires community members.
-Empower the community leader to dedicate 100 % of his time to community work including communication and development. 
 
-**Review comments**
-- (**Done**) important point is that community leader is excellent networker - not necessarily already networked
+Empower the community leader to dedicate 100 % of his time to community work including communication and development. 
 
 ## Resulting Context
 
 A community leader with the properties described above will lend a face and embody the companies commitment to InnerSource. It will make it more likely that other associates in his network will follow his lead and contribute to InnerSource. Over time, he will be able to build up a stable core team of developers and hence increase the chances of success for the InnerSource project. By convincingly a large enough audience within his company of the potential of InnerSource, he will make an important contribution to changing the company culture towards a community culture. 
```

Comment:
Testing testing; what if I didn't like this paragraph. Can I comment on it? Looks like it!